textattack/attack_recipes/cheng_2018_seq2sick.py

```diff
@@ -0,0 +1,38 @@
+"""
+    Cheng, Minhao, et al. 
+    
+    Seq2Sick: Evaluating the Robustness of Sequence-to-Sequence Models with 
+        Adversarial Examples
+    
+    ArXiv, abs/1803.01128.
+    
+    
+    This is a greedy re-implementation of the seq2sick attack method. It does 
```

Comment:
I think until we implement gradient descent we should name it seq2sick_black_box or something, since it's not really the same method while the other attack recipes are supposed to emulate as close as possible.

Reply:
ok good idea